templates/data/kyma-integration-data.yaml

```diff
@@ -505,6 +512,30 @@ templates:
                     - "request_small"
                     - "vm_job_template"
                     - "vm_job_labels_template"
+              - jobConfig:
+                  name: "pre-main-cluster-users-integration-k3d"
```

Comment:
How about post-main job? I think we should have one.

Reply:
@Tomasz-Smelcerz-SAP done